### PR TITLE
feat(viewports): Add support for custom pixel densities

### PR DESCRIPTION
**Pull request by brianokeefe**
This diff adds support for changing `deviceScaleFactor` for individual viewports. To set this value for a viewport, add an `@` with the desired scale to the viewport string. For example, to simulate a Retina display (2x), you can invoke `html-sketchapp` like so:

    html-sketchapp --viewports.Retina 1024x768@2 --url example.com

This also accepts floating point values. If no `@` is provided, then the scale factor defaults to 1.

Note: even without this diff, you can change `deviceScaleFactor` globally by passing a flag to the browser instance via puppeteer. The difference here is that this change allows different scales for individual viewports, which may be useful. That said, if you'd prefer to just rely on the browser flag for this behavior, I understand completely 😄 Let me know what you think, I'm happy to update the README if you do want to move forward with this change.

### CODE REVIEW

**Comment by markdalgleish** [comment]
This looks great. Would you also be able to add a simple test for this to `viewports.test.js`?

**Comment by brianokeefe** [comment]
Thanks! Done 😄

File: README.md [hunk 1]
```diff
@@ -110,6 +110,14 @@ $ html-sketchapp --viewports.Desktop 1024x768 --viewports.Mobile 320x568 --file
...
 If multiple screen sizes are provided, the viewport name will be being appended to all symbol and text style names. For example, `Button/Primary` will be exported as `Button/Primary/Desktop` and `Button/Primary/Mobile`.
 
+Optionally, you can set the pixel density for a given viewport by adding an `@` followed by the desired scaling factor to the end of the viewport's resolution. For example, you can simulate a 1.5x and 2x display like so:
+
+```bash
+$ html-sketchapp --viewports.HigherDensity 1024x768@1.5 --viewports.Retina 1024x768@2 --file sketch.html --out-dir dist
+```
+
+If no scaling factor is provided, a default of `1` will be used.
+
```
Comment (markdalgleish) [inline README.md]: Thanks so much for writing this documentation—it's spot on 👌

**Comment by markdalgleish** [comment]
Just published as [v0.3.0](https://github.com/seek-oss/html-sketchapp-cli/releases/tag/v0.3.0). Thanks so much for this addition, with such a simple API and perfect documentation too 👏👏

**Pull request merged**